audio-worklet/bitcrusher.html

```diff
@@ -13,46 +13,48 @@
 limitations under the License.
 -->
 
-<!-- 
-  This code illustrates a web audio bitcrusher implementation, a musical audio 
-  effect. The bitcrusher class defined in bitcrusher.js binds the script 
-  processor node's onaudioprocess method to a bitcrush function, which reduces 
-  the precision of individual samples and simulates a sample rate reduction by a
-  specified factor. These parameters can be modified as the sound source is 
-  running concurrently. 
+<!--
+  This code illustrates a web audio bitcrusher implementation, a musical audio
+  effect. There are two bitcrusher classes. One is defined in
+  bitcrusher-script-processor.js which binds the script
+  processor node's onaudioprocess method to a bitcrush function.
+  The other bitcrusher is defined in bitcrusher-audio-worklet.js and uses
+  the audio worklet node. The user can switch between the two as audio is
+  played. Both implementations enable a user to alter the bitDepth of individual
+  samples as well change the sample rate by a specified factor.
 -->
 <!DOCTYPE html>
 <html>
   <head>
     <title>Bitcrusher</title>
     <script src="src/bitcrusher-script-processor.js"></script>
-    <script src="src/source-controller.js"></script>
-    <script src="src/param-controller.js"></script>
+    <script src="lib/source-controller.js"></script>
+    <script src="lib/param-controller.js"></script>
     <script src="src/bitcrusher-demo.js"></script>
+    <script src="lib/audio-worklet-helper.js"></script>
   </head>
   <body>
     <h1>Bitcrusher: a lo-fi audio effect</h1>
     <p>
-      Hit play to start a song and listen to the distortive effect of 
+      Hit play to start a song and listen to the distortive effect of
       modifying the bit depth and sample rate reduction parameters.
-    </p> 
+    </p>
     <p>
       The bit depth slider controls the number of bits used to represent
       a given sample. The sample rate reduction slider affects the number
       of distinct samples in the signal. Increasing this value simulates
       sample rate reduction by making intervals of consecutive samples equal.
     </p>
     <div id="container"></div>
-    <script> 
-      // Default to 50% volume, maximum bit depth, and no sample rate reduction.
-      let bitDepth = 24.0;
-      let reduction = 1.0;
-      let gain = 0.5;
-
-      window.addEventListener('load', () => {
-          let controller = new BitcrusherDemo(
-              'container', bitDepth, reduction, gain);
-      });
+    <script>
+      function runBitcrusherDemo() {
+          window.audioWorklet.addModule('src/bitcrusher-audio-worklet.js')
+              .then(() => {
+                  const context = new AudioContext();
+                  let demo = new BitcrusherDemo(context, 'container');
+              });
+      }
+      AudioWorkletHelper.startDemo(runBitcrusherDemo);
```

Comment:
Doesn't this mean that you can't run the script processor demo anymore.  Is this really what we want?  I was kind of expecting to be able to run both (not at the same time of course).